src/components/LegoGallery/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import ThemedImage from '@theme/ThemedImage';
+import clsx from 'clsx';
+
+const STANDARDS = [
+  {
+    title: 'LSP0 - ERC725 Account',
+    link: './universal-profile/lsp0-erc725account',
+    lightImage: '/img/standards/legos/LSP0-lego.png',
+    darkImage: '/img/standards/legos/LSP0-lego.png',
```

Comment:
Having a dark Image would be great. Sharp white edges always look horrific in dark mode. Someone may have the time to go through all the images once. 

<img width="1038" alt="bricks" src="https://github.com/lukso-network/docs/assets/61689369/ac691a90-8197-456f-ac0a-5519fa2fd710">

If you can export the pictures with different backgrounds or transparency, I recommend doing so now (if you have the layout in [Mecabricks](https://www.mecabricks.com/en/workshop)). Otherwise, we might lose time later on.

Reply:
Agree on supporting the images for dark theme too here